### PR TITLE
Update remove-former-employee.md

### PR DESCRIPTION
**Pull request by msbemba**
Added Setup Call forwarding to people in your organization

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2435

File: microsoft-365/admin/add-users/remove-former-employee.md [hunk 1]
```diff
@@ -339,6 +339,12 @@ For more information about managing user licenses for Microsoft 365 for business
 When you remove a user's license from Office 365, the PSTN calling number associated with the user will be released. You can assign it to another user.
   
 If the user belongs to a queue group, they will no longer be a viable target of the call queue agents. So, we recommend also removing the user from the groups associated with the call queue. 
+
+## Setup Call forwarding to people in your organization
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/remove-former-employee.md]:
```suggestion
## Set up call forwarding to people in your organization
```

File: microsoft-365/admin/add-users/remove-former-employee.md [hunk 2]
```diff
@@ -339,6 +339,12 @@ For more information about managing user licenses for Microsoft 365 for business
...
+
+If you need to setup call forwarding for the terminated employees phone number, the call forwarding setting under calling policues can setup forwarding  where incoming calls can be forwarded to other users or can ring another person at the same time. For more information, See [Calling policies](https://docs.microsoft.com/microsoftteams/teams-calling-policy)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/remove-former-employee.md]:
```suggestion
If you need to set up call forwarding for the terminated employee's phone number, the call forwarding setting under calling policies can set up forwarding where incoming calls can be forwarded to other users or can ring another person at the same time. For more information, see [Calling policies in Microsoft Teams](https://docs.microsoft.com/microsoftteams/teams-calling-policy).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**